project/lesson4/final/index.html

```diff
@@ -21,7 +21,7 @@
   <link rel="stylesheet" href="http://weloveiconfonts.com/api/?family=zocial" />
 
   <!-- loads the Udacity Feedback extension -->
-  <meta name="udacity-grader" content="http://udacity.github.io/responsive-images/project/project-grader.json" unit-tests="http://udacity.github.io/responsive-images/project/project-grader.js">
+  <meta name="udacity-grader" content="project-grader.json" unit-tests="project-grader.js">
```

Comment:
@notetiene why do you want to move the test json into the zip files we give to students rather than keeping it external?